src/model/model.nit

```diff
@@ -627,7 +638,7 @@ class MClassDef
 	var in_hierarchy: nullable POSetElement[MClassDef] = null
 
 	# Is the definition the one that introduced `mclass`?
-	fun is_intro: Bool do return mclass.intro == self
+	fun is_intro: Bool do return isset mclass._intro and mclass.intro == self
```

Comment:
The trap with Ruby syntax is that new users coming from Ruby will think of it as the standard way and not the "pretty rare and not so elegant way".